### ISSUE TITLE
Landing page for mobile devices

### ISSUE THREAD

**Issue by allensam** [youngstartupsio/Yeezy-Codes#22]
I would like there to be a landing page for smaller devices (ones without keyboards) so that we can tell what the project is all about and maybe have some mobile friendly examples so they'll be convinced to get on a laptop/desktop and try it out!

**Comment by dinubs** [comment]
Make the whole site responsive, don't make different pages just for mobile/desktop

**Comment by allensam** [comment]
Yeah true that someone take this issue on pls I'm making some badass responsive angular ;)
